homework/12_V/25_Stefan_Aleksandrov/areColliding.py

```diff
@@ -0,0 +1,24 @@
+import math
+
+def areColliding(x1, y1, x2, y2, r1, r2):
```

Comment:
The homework was to determine if two circles intersect with each other and if so how exactly - touching, intersecting at two points, one withing the other, or equal. You compute if one is touching or inside the other which is a slightly different task.